mscore/inspector/voicing_select.ui

```diff
@@ -118,12 +115,12 @@
      </item>
      <item>
       <property name="text">
-       <string>Stop at measure</string>
+       <string>Stop at measure end</string>
```

Comment:
This is what is supposed to to as per the code

Reply:
Why not "Until Measure End," for consistency with "Until Next Chord Symbol"?

Reply:
True, this is better. And shorter. Or better because shorter ;-)